setup.cfg

```diff
@@ -1,24 +1,25 @@
 [metadata]
 name = scib
 version = 0.2.0
-description = Benchmark tools for single cell data integration
+description = Evaluating single-cell data integration methods
 long_description = file: README.md
 long_description_content_type = text/markdown
-author = Malte Luecken, Maren Buettner, Daniel Strobl, Michaela Mueller
-author_email = malte.luecken@helmholtz-muenchen.de, michaela.mueller@helmholtz-muenchen.de
+author = Malte D Luecken, Maren Buettner, Daniel C Strobl, Michaela F Mueller
+author_email = malte.luecken@helmholtz-muenchen.de
```

Comment:
you can keep your helmholtz email on there if you want. And if you amend... add dots to the initials please ^^